### ISSUE TITLE
Can I add "returnTo" to url parameters: like /login?return_url=secretpage ?

### ISSUE THREAD

**Issue by IvanRave** [jaredhanson/connect-ensure-login#10]
> Middleware sets session.returnTo to /settings
> And redirect to /login

Can I add "returnTo" to url parameters: like /login?return_url=secretpage ?

**Comment by nxz91** [comment]
I agree this should be possible. If it's not part of the URL the login URL will not be shareable / bookmarkable as a point of entry.
